index.html

```diff
@@ -8,6 +8,76 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <h1>Hello to noone</h1>
+  <div class="right-button">
+    <div class="signin-text">
+      <div>Already have an Intuit account?</div>
+      <a href="">I forgot my User ID or Password</a>
+    </div>
+    <button class="signin-btn">
+      <img src="/assets/images/icon-whitelock@3x.png" alt="lock-icon">
+      Sign In
+    </button>
+  </div>
+
+  <main class="main-wrapper">
+    <div class="main-logo-box">
+      <img src="/assets/images/intuit_logos_new.png" alt="main-logo" class="main-logo">
+    </div>
+
+    <div class="product-logos">
+      <img src="/assets/images/mint_logos_new.png" alt="mint">
+      <img src="/assets/images/qb_logos_new.png" alt="qb">
+      <img src="/assets/images/turbo_logos_new.png" alt="turbo">
```

Comment:
You don't need this first forward slash since the assets directory it's the same one as the index.html. Should be src="assets/images/... for these 3 img tags.